pkg/client/factory.go

```diff
@@ -151,6 +154,8 @@ func (f *factory) KubebuilderClient() (kbclient.Client, error) {
 
 	scheme := runtime.NewScheme()
 	velerov1api.AddToScheme(scheme)
+	k8scheme.AddToScheme(scheme)
```

Comment:
Were these changes necessary as part of the CI fix?

Reply:
Good catch @zubron. No, this was supposed to be part of another branch for an incoming change, but not this one. Thank you, reverted.